### PR TITLE
feat: operator title

### PR DESCRIPTION
**Pull request by shawnyama**
# Description

- Created `tera-operator-title` to display title of operator (typically the asset name)

### CODE REVIEW

File: packages/client/hmi-client/src/workflow/operator/tera-operator-title.vue [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<template>
+	<h5 v-if="title">{{ title }}</h5>
```
Comment (YohannParis) [inline packages/client/hmi-client/src/workflow/operator/tera-operator-title.vue:2]: I wonder if a slot would be better, as this is the only thing.

**Review by YohannParis — COMMENTED** [review]
Why this is not part of `tera-header`???

**Comment by shawnyama** [comment]
> Why this is not part of `tera-header`???

It's separate from the header. The header has its own title which is operation type. This title is for showing the asset type when applicable.

File: packages/client/hmi-client/src/workflow/operator/tera-operator-title.vue (hunk 1 above)
Comment (shawnyama) [reply packages/client/hmi-client/src/workflow/operator/tera-operator-title.vue:2]: What would the slot have other than an `h5`? I feel like this doesn't have to be a component.

**Review by YohannParis — APPROVED** [review]

**Pull request merged**